src/hemi-sepolia.tokenlist.json

```
@@ -0,0 +1,65 @@
+{
+  "name": "Hemi Token List",
+  "timestamp": "2024-05-07T22:03:02.686Z",
+  "version": {
+    "major": 1,
+    "minor": 0,
+    "patch": 0
+  },
+  "tokens": [
+    {
+      "address": "0xec46E0EFB2EA8152da0327a5Eb3FF9a43956F13e",
+      "chainId": 743111,
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/ethereum-optimism/ethereum-optimism.github.io/master/data/DAI/logo.svg",
+      "name": "Hemi Tunneled DAI",
+      "symbol": "DAI",
+      "extensions": {
+        "birthBlock": 71589,
+        "bridgeInfo": {
+          "1": {
+            "tokenAddress": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+            "originBridgeAddress": "0xc94b1BEe63A3e101FE5F71C80F912b4F4b055925",
+            "destBridgeAddress": "0x4200000000000000000000000000000000000010"
+          }
+        }
+      }
+    },
```

sepolia data is missing, and I think the bridges addresses are actually from Sepolia cc @gabmontes 

```suggestion
    {
      "address": "0xec46E0EFB2EA8152da0327a5Eb3FF9a43956F13e",
      "chainId": 743111,
      "decimals": 18,
      "logoURI": "https://raw.githubusercontent.com/ethereum-optimism/ethereum-optimism.github.io/master/data/DAI/logo.svg",
      "name": "Hemi Tunneled DAI",
      "symbol": "DAI",
      "extensions": {
        "birthBlock": 71589,
        "bridgeInfo": {
          "1": {
            "tokenAddress": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
          },
          "11155111": {
            "originBridgeAddress": "0xc94b1BEe63A3e101FE5F71C80F912b4F4b055925",
            "destBridgeAddress": "0x4200000000000000000000000000000000000010"
          }
        }
      }
    },
```

---

Fixed